src/components/LeftSidebar/ConversationsList/AppContentListItem/AppContentListItem.vue

```diff
@@ -21,40 +21,42 @@
 
 <template>
 	<nav-element
-		:class="{ 'active' : isActive }"
-		v-bind="navElement"
-		href="#"
-		class="acli"
-		@click="onClick">
-		<!-- default slot for avatar or icon -->
-		<slot name="icon" />
-		<div class="acli__content">
-			<div class="acli__content__line-one">
-				<span class="acli__content__line-one__title">
-					{{ title }}
-				</span>
-				<Actions
-					v-if="hasActions"
-					menu-align="right"
-					class="acli__content__line-one__actions">
-					<slot name="actions" />
-				</Actions>
-				<span
-					v-if="{hasDetails}"
-					class="acli__content__line-one__details">
-					{{ details }}
-				</span>
+		v-bind="navElement">
+		<a
+			:class="{ 'active' : isActive }"
+			href="#"
+			class="acli"
+			@click="onClick">
+			<!-- default slot for avatar or icon -->
+			<slot name="icon" />
+			<div class="acli__content">
+				<div class="acli__content__line-one">
+					<span class="acli__content__line-one__title">
+						{{ title }}
+					</span>
+					<Actions
```

Comment:
Actions cannot be inside the link I think, I can already see issues with it :thinking: 
The link should probably only cover the icon and titles?

Reply:
The inner <a> (actions) seems to have precedence over the grandparent <a>, so far I tested this with chrome and firefox, with clicking, tapping and keyboard navigation and it all works good. I'm looking for documentation on this but I can't find much.